Phone book/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import "fmt"
+
+func List(){
+	file, err := os.Open("phone book.json")
```

Comment:
Since you aren't decoding JSON, you could just print the whole file on the screen with this command:

```go
data, err := os.ReadFile("phone book.txt")
if err != nil {
  log.Fatal(err)
}
fmt.Println(string(data))
```